### ISSUE TITLE
Snow melts in back-converted worlds

### ISSUE THREAD

**Issue by carl-sc** [Amulet-Team/Amulet-Map-Editor#532]
**Describe the bug**
When converting a world from 1.14 or later to 1.13.2 or earlier, snow (both snow blocks and layers) will melt when the converted world is opened. Only affects snow in the copied areas. 

**To Reproduce**
A relatively simple example of this issue can be found with world seed 200, which drops the user into the Snowy Plains biome. 
1. Create a new world in 1.14 with seed 200. 
2. If interested, add some snow blocks. 
3. Create a new world in 1.13 with seed 200.
4. Convert the first world into the second. 
5. Open the converted world.

**Expected behavior**
Both snow blocks and partial snow layers should probably be preserved.

**Screenshots**
See [this album](https://imgur.com/a/gcnt4YE). 

**Desktop (please complete the following information):**
 - OS: Windows 8 or 10

**Additional context**
Originally found in converting 1.16.5 worlds to 1.12.2, where snow not only is destroyed randomly but also slows down the world significantly by melting into snowballs. 

The only thing in the 1.14 version updates that I could find that indicates a change to snow behavior is one [bug fix](https://bugs.mojang.com/browse/MC-88097), which may be relevant, although it only deals directly with snow blocks, not snow layers. My only other hypotheses are (1) that it might have something to do with the `BlockStateTag`, which was new in 1.14, or (2) that the addition of snowy tundra villages might cause problems with how cold biomes are treated.

**Comment by gentlegiantJGC** [comment]
I presume you are talking about Java edition.
Could you check the biome in that area before and after conversion.

**Comment by carl-sc** [comment]
Yes, sorry for missing that. The biome remains `minecraft:snowy_tundra` in both. The same issue is observed in snowy beach (in 1.14; in 1.13.2 it's cold beach), where both snow and ice melt.
